### PR TITLE
hotdogb: init crDroid 11

### PR DESCRIPTION
**Pull request by frazerecho**

### CODE REVIEW

File: hotdogb.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"device": "OnePlus 7T",
+			"filename": "crDroidAndroid-15.0-20241215-hotdogb-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/hotdogb/11.x/crDroidAndroid-15.0-20241215-hotdogb-v11.0-BETA.zip/download",
+			"timestamp": 1734264573,
+			"md5": "704623ed66c58b1d7e2dce2c499928e7",
+			"sha256": "f329cedd004356b1373f2e98fe15a2d342031d06b0a8ce46c47275e6d7101a3b",
+			"size": 1395671312,
+			"version": "11.0",
+			"buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/4467671/",
```
Comment (gwolf2u) [inline hotdogb.json:15]: please update forum to new template
https://github.com/crdroidandroid/crdroid_features/blob/15.0/ForumPostTemplate
also include follow back donation url 
ofc can have your own donate page, but include ours also please

File: hotdogb_changelog.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Highlights & Device Specific Changes:
+Build type: BETA
+Device: OnePlus 7T (hotdogb)
+Device maintainer: Hasan Okarci (frazerecho)
+
+
```
Comment (gwolf2u) [inline hotdogb_changelog.txt:6]: can remove one or two blank spaces here

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: hotdogb_changelog.txt (hunk 2 above)
Comment (frazerecho) [reply hotdogb_changelog.txt:6]: Done

File: hotdogb.json (hunk 1 above)
Comment (frazerecho) [reply hotdogb.json:15]: Done as per the template. Kindly check as I'm confused about the donation thing.

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**